### PR TITLE
Create Command_kick.java

### PR DESCRIPTION
**Pull request by DUFCLiam**

### CODE REVIEW

**Comment by DUFCLiam** [comment]
@aggelosQQ You requested this :)

**Comment by aggelosQQ** [comment]
Yes, I did.
@LegendIsAwesomes @RobinGall2910 @OxLemonxO

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_kick.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    sender.sendMessage(TFM_Command.MSG_NO_PERMS);
+                }
+                
+                if (!TFM_AdminList.isSuperAdmin(sender_p))
+                {
+                    if (args.length < 1) {
+                        
+                    sender_p.kickPlayer(ChatColor.RED + "You have been kicked by: " + sender.getName() + "-" + " No reason given");
+                    }
+                        }
```
Comment (OxLemonxO) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_kick.java]: Formatting.

**Comment by DUFCLiam** [comment]
I can no longer edit this.

**Pull request closed without merging**